apps/ios/Shared/Views/Helpers/ImagePicker.swift

```diff
@@ -9,6 +9,64 @@
 import SwiftUI
 import PhotosUI
 
+func dropPrefix(_ s: String, _ prefix: String) -> String {
+    s.hasPrefix(prefix) ? String(s.dropFirst(prefix.count)) : s
+}
+
+func dropImagePrefix(_ s: String) -> String {
+    dropPrefix(dropPrefix(s, "data:image/png;base64,"), "data:image/jpg;base64,")
+}
+
+func resizeAndCrop(_ image: UIImage, to newSize: CGSize) -> UIImage {
+    let format = UIGraphicsImageRendererFormat()
+    format.scale = 1.0
+    format.opaque = true
+    return UIGraphicsImageRenderer(bounds: CGRect(origin: .zero, size: newSize), format: format).image { _ in
+        let size = image.size
+        let hScale = newSize.height / size.height
+        let vScale = newSize.width / size.width
+        let scale = max(hScale, vScale) // scaleToFill
+        let resizeSize = CGSize(width: size.width * scale, height: size.height * scale)
+        var middle = CGPoint.zero
+        if resizeSize.width > newSize.width {
+            middle.x -= (resizeSize.width - newSize.width) / 2
+        } else if resizeSize.height > newSize.height {
+            middle.y -= (resizeSize.height - newSize.height) / 2
+        }
+        image.draw(in: CGRect(origin: middle, size: resizeSize))
```

Comment:
nit: I find origin being the middle confusing. Does the image span -width/2, width/2?
I would have origin as bottom left corner but I may be silly

Reply:
it works, I'd rather keep as is :)